packages/blade/src/components/Input/BaseInput/baseInputStyles.ts

```diff
@@ -117,6 +117,7 @@ export const getBaseInputStyles = ({
         ? makeSpace(theme.spacing[3])
         : makeSpace(theme.spacing[4]),
     textAlign,
+    width: '100%',
```

Comment:
The individual inputs need to take up complete available width otherwise the layout messes up. Reference conversation: https://razorpay.slack.com/archives/G01B3LQ9H0W/p1663751935820919

Reply:
No change on the existing inputs as per chromatic. Did a manual check on existing stories for other inputs and things seem okay